app/config.js

```diff
@@ -84,3 +93,18 @@ exports.getPlugins = function () {
     localPlugins: cfg.localPlugins
   };
 };
+
+exports.window = {
+  get () {
+    let position = winCfg.get('windowPosition');
+    let size = winCfg.get('windowSize');
+    return {
+      position: (position !== undefined) ? position : winCfg.store.position,
+      size: (size !== undefined) ? size : winCfg.store.size
```

Comment:
You've set `defaults` in the Config constructor, so `position` and `size` will never be `undefined`.